modules/custom/social_scroll/src/SocialScrollOverride.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Drupal\social_scroll;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+
+/**
+ * Class SocialScrollOverride.
+ *
+ * @package Drupal\social_scroll
+ */
+class SocialScrollOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * The config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * The SocialScrollManager manager.
+   *
+   * @var \Drupal\social_scroll\SocialScrollManager
+   */
+  protected $socialScrollManager;
+
+  /**
+   * Constructs the configuration override.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory.
+   * @param \Drupal\social_scroll\SocialScrollManager $social_infinite_scroll_manager
+   *   The SocialScrollManager manager.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, SocialScrollManager $social_infinite_scroll_manager) {
+    $this->configFactory = $config_factory;
+    $this->socialScrollManager = $social_infinite_scroll_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+    $enabled_views = $this->socialScrollManager->getEnabledViewIds();
+
+    foreach ($enabled_views as $key => $status) {
+      $config_name = $this->socialScrollManager->getConfigName($key);
+
+      if (in_array($config_name, $names)) {
+        $current_view = $this->configFactory->getEditable($config_name);
+        $displays = $current_view->getOriginal('display');
+        $pages = [];
+
+        foreach ($displays as $id => $display) {
+          if (!isset($display['display_options']['pager']) || $display['display_plugin'] === 'block') {
+            continue;
+          }
+          $pages[] = $id;
```

Comment:
```suggestion
          if (isset($display['display_options']['pager']) && $display['display_plugin'] !== 'block') {
            $pages[] = $id;
          }
```

Reply:
Changed.

Reply:
Operation between conditions should be **AND** instead of **OR**. Also, the value from the **display_plugi**" key should be equal to the **block** value